src/main/resources/assets/styles/dialog/archive-dialog.less

```diff
@@ -10,9 +10,15 @@
         display: flex;
         flex-direction: column;
 
-        .archive-item-viewer {
+        .archive-dialog-item-list {
+          overflow: auto;
+          max-height: 40%;
           flex-shrink: 0;
-          padding: 4px 0;
+
+          .archive-item-viewer {
+            flex-shrink: 0;
+            padding: 4px 0;
+          }
```

Comment:
@ashklianko shouldn't we have common styles/scrolling behaviour for item list in all dialogs?

Reply:
Aligned styling with common used in dialogs